### ISSUE TITLE
How can i pass endpointName & custom container env variables on HostingDeployment ?

### ISSUE THREAD

**Issue by nandhyala** [aws/amazon-sagemaker-operator-for-k8s#146]
@surajkota 

How can i pass endpointName & custom container env variables on HostingDeployment ?

**Comment by surajkota** [comment]
Environment variables can be injected per model container hence they are part of models field in the spec. Here is a sample to look at how `MY_VAR` and `MY-ENDPOINT-NAME` for endpoint name and custom env variables are passed in the spec below. Deterministic endpoint name is not available on the latest release v1.1.0. It will be available once we cut next release
 
```
apiVersion: sagemaker.aws.amazon.com/v1
kind: HostingDeployment
metadata:
  name: xgboost-hosting
spec:
  region: us-west-2
  endpointName: MY-ENDPOINT-NAME
  productionVariants:
    - variantName: AllTraffic
      modelName: xgboost-model
      initialInstanceCount: 1
      instanceType: ml.r5.large
      initialVariantWeight: 1
  models:
    - name: xgboost-model
      executionRoleArn: arn:aws:iam::169544399729:role/m-sagemaker-role
      containers:
        - containerHostname: xgboost
          modelDataUrl: s3://surakota-kfp-test-data/inference/xgboost-mnist/model.tar.gz
          image: 433757028032.dkr.ecr.us-west-2.amazonaws.com/xgboost:latest
          environment:
            - name: MY_VAR
              value: my_env_value
      primaryContainer: xgboost
  tags:
      - key: name
        value: sagemaker-k8s-operator tag test
```

**Comment by surajkota** [comment]
Please re-open the issue if you have more questions

Thank you
